tests/config/semicolon-removal-after-ternary-operator.cfg

```diff
@@ -0,0 +1 @@
+mod_remove_extra_semicolon = true
```

Comment:
BTW, this file is no longer needed.

Reply:
yeah, totally forgot to remove it
thanks